<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>selectCourseManage</class>
 <widget class="QWidget" name="selectCourseManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>311</width>
     <height>531</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>选课信息</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <column>
       <property name="text">
        <string>学号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>课程号</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="export_button">
      <property name="text">
       <string>导出为Excel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>80</y>
     <width>273</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,3,3">
    <item>
     <widget class="QLabel" name="currentSelectedLabel">
      <property name="text">
       <string>当前选中单元格数据</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>学号：  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="sNo_lineEdit"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>课程号：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="cId_lineEdit"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_5" native="true">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>270</y>
     <width>231</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QPushButton" name="searchBySNo_button">
      <property name="text">
       <string>根据学号查询</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="searchByCId_button">
      <property name="text">
       <string>根据课程号查询</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="listAll_button">
      <property name="text">
       <string>显示所有选课信息</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="add_button">
      <property name="text">
       <string>添加选课信息</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="delete_button">
      <property name="text">
       <string>删除选择信息</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
